src/fft.js

```diff
@@ -467,9 +467,7 @@ define(function (require) {
    *                               Defaults to 0.8.
    */
   p5.FFT.prototype.smooth = function(s) {
-    if (s) {
-      this.smoothing = s;
-    }
+    this.smoothing = s;
     this.analyser.smoothingTimeConstant = s;
```

Comment:
For parity reasons w/ properties created via operator `=`, when using **defineProperty()** to create an accessor property, include as well the descriptors _configurable_ & _enumerable_ set as `true`:

https://developer.Mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty#Adding_properties_and_default_values

```js
p5.FFT = function (smoothing, bins) {
  this.analyser.smoothingTimeConstant = +smoothing || 0.8;

  Object.defineProperty(this, 'smoothing', {
    get: function ()  { return this.analyser.smoothingTimeConstant; },
    set: function (s) { this.analyser.smoothingTimeConstant = +s || 0.8; },
    configurable: true, enumerable: true
  });
};
```